src/actions/WindowActions.js

```diff
@@ -536,7 +537,7 @@ export function initWindow(windowType, docId, tabId, rowId = null, isAdvanced) {
               includedTabsInfo: {},
               scope: 'master',
               saveStatus: { saved: true },
-              standardActions: {},
```

Comment:
I think the problem was here - it was a plain object, so `{}.has` was `undefined` instead of being [`Set.prototype.has`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Set/has). [`Set`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Set) is natively supported from the global object

Reply:
It was even weirder. Cause when I logged `Set` it was printed as undefined initially. Maybe it's some webpack bullshit.